web-ui/src/main.tsx

```diff
@@ -1,10 +1,14 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.tsx'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.tsx";
+import "./index.css";
```

Comment:
I think this is a linter thing, we should sync on what linter we want to use.

Reply:
I will create a ticket to add Airbnb linter